website/pages/en/developing/assemblyscript-api.mdx

```diff
@@ -280,6 +280,20 @@ As the entity may not exist in the store yet, the `load` method returns a value
 
 > **Note:** Loading entities is only necessary if the changes made in the mapping depend on the previous data of an entity. See the next section for the two ways of updating existing entities.
 
+#### Looking up entities created withing a block
```

Comment:
should I also add the version of `graph-node` and `graph-cli` this feature is available?

Reply:
I think it good to mention this somewhere

Reply:
Agree, something like "This helper is generated by `graph-cli` for all entities as of v0.x.x, and subgraphs using this functionality can be deployed to Graph Node from v0.31.0."